jmclient/jmclient/blockchaininterface.py

```diff
@@ -167,22 +180,16 @@ def __init__(self, jsonRpc, network):
         actualNet = blockchainInfo['chain']
 
         netmap = {'main': 'mainnet', 'test': 'testnet', 'regtest': 'regtest'}
-        if netmap[actualNet] != network:
+        if netmap[actualNet] != network and \
+                (not (actualNet == "regtest" and network == "testnet")):
+            #special case of regtest and testnet having the same addr format
             raise Exception('wrong network configured')
 
-    def get_block(self, blockheight):
```

Comment:
Why remove this function? So far the only use case was scanning the blockchain in things like coinswap or scanning for coinjoins, i.e. only in applications outside joinmarket; but it was useful for that.

Reply:
I didn't see it used anywhere in the codebase. I honestly thought it was a leftover function from some earlier code that was deleted, since it seemed quite simple. I'll return the function if you think that's right.

Reply:
Yes, please do return it; I see no harm in having it (not to say it doesn't need improvement somehow; haven't thought about it for a while).

Reply:
Done